proto/eth/v1alpha1/validator.proto

```diff
@@ -220,6 +221,21 @@ service BeaconNodeValidator {
             body: "*"  
         };
     }
+
+    // Retrieves a sync committee message block root to be signed over as part of sync committee duty.
+    rpc GetSyncMessageBlockRoot(google.protobuf.Empty) returns (SyncMessageBlockRootResponse) {
```

Comment:
Discussed above, that we should instead maybe enforce by slot rather than head root.